utils/report.c

```diff
@@ -512,7 +515,7 @@ void report_diff_nodes(struct rb_root *orig_root, struct rb_root *pair_root,
 void destroy_diff_nodes(struct rb_root *diff_root)
 {
 	struct rb_node *n = rb_first(diff_root);
-	
+
```

Comment:
please do not change here.

Reply:
Original code has just space in empty line like below. it's from master branch.
Do you really want to keep this space?

```
514 >-------struct rb_node *n = rb_first(diff_root);$
515 >-------$
516 >-------while (n) {$
```

Reply:
No, please remove the spaces.